app/controllers/editorial/nodes_controller.rb

```diff
@@ -101,8 +96,7 @@ def configure_defaults!
 
     def try_save
       return false unless @form.validate(params.require(:node).permit!)
```

Comment:
Calling `@form.validate` will return a boolean expression with the result of the validation, so this method is probably unnecessary.